.github/sync-repo-settings.yaml

```diff
@@ -0,0 +1,30 @@
+# https://github.com/googleapis/repo-automation-bots/tree/main/packages/sync-repo-settings
+# Rules for main branch protection
+branchProtectionRules:
+# Identifies the protection rule pattern. Name of the branch to be protected.
+# Defaults to `main`
+- pattern: main
+  requiresCodeOwnerReviews: true
+  requiresStrictStatusChecks: true
+  requiredStatusCheckContexts:
+    - 'cla/google'
+    - 'OwlBot Post Processor'
+    - 'Kokoro'
+    - 'Kokoro SQLAlchemy compliance'
+    - 'Kokoro prerelease'
```

Comment:
Let's not block PR merges due to prerelease failures. I do want to know about them and get flakeybot issues, though.